influx_logging/handler.py

```diff
@@ -22,7 +22,7 @@
     data, text = bytes, str
 else:
     data, text = str, unicode
-    
+
```

Comment:
My editor is configured to remove extraneous whitespace. I hope that's alright! 🤔